bin/memcacheSync.php

```diff
@@ -3,18 +3,19 @@
 
 
 // Check that the memcache library is enabled
-if (!class_exists('Memcache')) {
-    echo("Error: the memcache library appears to be unavailable.\n");
-    echo("\n");
-    echo("This is most likely because PHP doesn't load it for the command line\n");
-    echo("version. You probably need to enable it somehow.\n");
-    echo("\n");
-    if(is_executable('/usr/sbin/phpenmod')) {
-        echo("It is possible that running the following command as root will fix it:\n");
-        echo(" phpenmod -s cli memcache\n");
-    }
-
-    exit(1);
+if(!class_exists('Memcache') && !class_exists('Memcached')) {
+	echo("Error: the memcache and memcached libraries appear to be unavailable.\n");
+	echo("\n");
+	echo("This is most likely because PHP doesn't load it for the command line\n");
+	echo("version. You probably need to enable one of them somehow.\n");
+	echo("\n");
+	if(is_dir('/etc/php5/cli/conf.d')) {
```

Comment:
It _is_ CLI-configuration as this is an executable script I think?

It looks like a merge error, since this is the original advice that was updated in de972b674829d24ee6767d2b8fe0be99e686ef0d. I can reapply that commit later.

Nonetheless it remains an heuristic that tries to dole out some advice where it can, it does not have the ambition to support every OS and configuration. Whether that is useful is up for debate.